smsdk/client.py

```diff
@@ -545,3 +545,32 @@ def create_share_link(
         return dataViz(self.session, base_url).create_share_link(
             *args, assets, chartType, yAxis, xAxis, model, time_selection, **kwargs
         )
+
+    def get_raw_data(
+        self,
+        raw_data_table,
+        normalize=True,
+        fields=[],
+        time_selection=one_day_relative,
+        limit=400,
+        offset=0,
+        *args,
+        **kwargs,
+    ):
+        raw_data = smsdkentities.get("raw_data")
+        base_url = get_url(
+            self.config["protocol"], self.tenant, self.config["site.domain"]
+        )
+        select = []
+        if len(fields) > 0:
+            for field in fields:
+                dict_ = {}
+                dict_["name"] = field
+                select.append(dict_)
```

Comment:
Less instructions / steps.
```suggestion
        select = [
            {
                "name": field,
            }
            for field in fields
        ]
```

Reply:
Done!